packages/odata-service-inquirer/src/types.ts

```diff
@@ -330,6 +330,10 @@ export type ServiceSelectionPromptOptions = {
      * This is used to indicate that the service does not support collaborative draft.
      */
     showCollaborativeDraftWarning?: boolean;
+    /**
+     * A list of service ids used to filter the catalog results
```

Comment:
Can we add an example here as `service id` is not a property thats specified in the returned service object so a dev coming to this may not know what that it?